policy/testdata/README.md

```diff
@@ -0,0 +1,170 @@
+# CEL Policy
+
+The Common Expression Language (CEL) supports simple expressions: no variables, functions, or modules. However, CEL expression graphs can be composed together, allowing for reuse and development clarity which is not otherwise possible within CEL.
+
+To address this case, we're introducing the CEL Policy format which is fully runtime compatible with CEL. All of the same performance and safety hardening guarantees which apply to CEL also apply to CEL Policy. The net effect is significantly improved authoring and testability. The YAML-based policy format is easily extensible and inspired by [Kubernetes Admission Policy](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/) with CEL.
+
+## Policy Language
```

Comment:
Originally the top level rule was plural, yeah? Is it always single now?

Reply:
It's been single ever since we introduced nesting support.